plat/kvm/x86/multiboot.S

```diff
@@ -77,10 +78,18 @@ ENTRY(_multiboot_entry)
 	cmpl	$MULTIBOOT_BOOTLOADER_MAGIC, %eax
 	jne	no_multiboot
 
-	movl	$lcpu_boot_startup_args, %edi	/* startup args for boot CPU */
+	/* Hardcoding for now I guess... */
+	movl    $0x00100000, %edi
```

Comment:
These 3 lines I do not like, maybe some way to add an additional configuration option for the load address in the linker script, but while allowing three different versions (e.g. if a user wants to build KVM, Xen and linuxu images with the same configurations). So RFC.

Reply:
I would leave them as such for the time being. Let's have something working first and wait for the time when a configurable setup is required.